packages/wp-source/src/actions.ts

```diff
@@ -11,19 +11,23 @@ import { ErrorData } from "@frontity/source/types/data";
 import { ServerError } from "@frontity/source";
 
 const actions: WpSource["actions"]["source"] = {
-  fetch: ({ state, libraries }) => async link => {
+  fetch: ({ state, libraries }) => async params => {
     const { source } = state;
 
     const { handlers, redirections } = libraries.source;
 
+    // Get params
```

Comment:
I believe that writing explicit `if...else` is more readable than a ternary in most cases, even if it covers more lines. Something like:
```js
let link;
let force = false;
if (typeof params === "string") {
  link = params;
} else {
  link = params.link;
  force = params.force;
}
```
It also makes it more easily extensible in the future, because you only need to add another `else if {}` if there is another case to handle.

Your code is fine, of course, you **don't** need to change anything. I'm just using this as an opportunity to share an opinion 🙂